src/test/scala/biowdl/test/TestPipelineFunctionalTest.scala

```diff
@@ -21,11 +21,6 @@
 
 package biowdl.test
 
-import nl.biopet.utils.biowdl.samples.{Wgs1PairedEnd, Wgs2PairedEnd}
-
-class TestPipelineFunctionalTest
-    extends TestPipelineSuccess
-    with Wgs1PairedEnd
-    with Wgs2PairedEnd {
+class TestPipelineFunctionalTest extends TestCompareGffSuccess {
```

Comment:
This should extend some real annotations, like `Ensembl87` and provide a real/large `sampleGtf`.

Reply:
We do not have a real/large sampleGtf yet.
We better include this with the functional test in an overarching multisample pipeline. Where we can let the big gtf be assembled from raw data.